apps/integrations/src/core/webhook/credentials.ts

```diff
@@ -0,0 +1,85 @@
+import { AuthCredentials } from "core/authentication/types";
+import { WebhookAuthentication } from "./subscribe/types";
+import { Service } from "core/service/types";
+
+let cachedClient: any = undefined;
+async function getNangoClient() {
+  const Nango = (await import("@nangohq/node")).Nango;
+
+  if (cachedClient !== undefined) {
+    return cachedClient as InstanceType<typeof Nango>;
+  }
+
+  cachedClient = new Nango({
+    host: process.env.NANGO_HOST,
+    secretKey: process.env.NANGO_SECRET_KEY,
+  });
+  return cachedClient as InstanceType<typeof Nango>;
+}
+getNangoClient();
```

Comment:
@ericallam I had to do this because Nango doesn't have CJS at the moment. I've requested it.